qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -34,10 +42,16 @@
         - with_repetition:
             repeat_times = 300
         - with_reboot:
+            sub_type_after_plug = boot
             sub_type_after_unplug = boot
             reboot_method = shell
         - with_shutdown:
-            sub_type_after_unplug = shutdown
+            variants:
+                - after_plug:
+                    sub_type_after_plug = shutdown
+                    sub_type_before_unplug = shutdown
```

Comment:
Not good to define sub_test "shutdown" before unplug, suggest to find a better way.